wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,281 @@
+# Scroll View
+
+## Summary
+
+This spec proposes to add an experimental native ScrollView component to Electron.
+
+## Platforms
+* macOS 
+* Windows
+* Linux
+
+## Impetus
+
+It is currently not possible to add scrolling functionality around BrowserViews:
+
+* Capture mouse / touch events
```

Comment:
I don't see any new events for capturing e.g. a click on a view. Is that out of scope for this?

Reply:
It is out of scopefor this spec; we extended the implementation for such functionalities.